binaries/org.eclipse.swt.win32.win32.x86_64/build.properties

```diff
@@ -35,8 +35,14 @@ source.. = \
 	../../bundles/org.eclipse.swt/Eclipse SWT Browser/win32,\
 	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/win32,\
 	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/common
+source.test = \
```

Comment:
Test folders should usually not be part of the `source.xxx` /  `bin.xxx` of PDE...

Reply:
I think I am not yet sure how to properly make Tycho compile the test classes. With this configuration, Tycho properly generates test classes in a separate output folder to be used for executing the maven-surefire-plugin. What would be the proper approach to achieve that?

Reply:
it will jsut work without that, as it is configured in the `.classpath` already.

Reply:
That's correct. Only the .classpath configuration is currently not considered because the .classpath file is not considered due to being a linked resource.